### DISCUSSION TITLE
Configurar envio de email pela Hostgator

### DISCUSSION THREAD

**Discussion by matheusneves2023** [RamonSilva20/mapos#2151] [Q&A]
Boa noite amigos,
Alguem conseguiu configurar o envio de emails pela Hostgator?
Aqui só fica Pendente.

Ja fiz algumas configurações mas sem sucesso...

Desde já agradeço

**Comment by GiovaniRio** [comment]
Estou com este problema. Conseguiu resolver? Obrigado

**Comment by itaymbere** [comment]
Maybe this link will help you @matheusneves2023 @GiovaniRio 
[https://www.hostgator.com/help/article/email-connection-settings](https://www.hostgator.com/help/article/email-connection-settings)
